### PR TITLE
Add rule to convert short template notation to bundle notation

### PR DESCRIPTION
**Pull request by arnaubuch**
Converts short Twig template notation (deprecated in 5.0) for bundle notation with `@` prefix.

Needed this rule for my own purposes but tought I'd share for anyone that might need it.

### CODE REVIEW

File: src/Rector/MethodCall/ConvertRenderTemplateShortNotationToBundleSyntaxRector.php
Comment (johanadivare) [inline src/Rector/MethodCall/ConvertRenderTemplateShortNotationToBundleSyntaxRector.php:24]:
```suggestion
```

File: src/Rector/MethodCall/ConvertRenderTemplateShortNotationToBundleSyntaxRector.php
Comment (johanadivare) [inline src/Rector/MethodCall/ConvertRenderTemplateShortNotationToBundleSyntaxRector.php:20]: Maybe you can also add a `@see https://github.com/symfony/symfony/pull/21035`

File: src/Rector/MethodCall/ConvertRenderTemplateShortNotationToBundleSyntaxRector.php
Comment (johanadivare) [inline src/Rector/MethodCall/ConvertRenderTemplateShortNotationToBundleSyntaxRector.php:78]: I would check this before the `ThisType` check above

File: src/Rector/MethodCall/ConvertRenderTemplateShortNotationToBundleSyntaxRector.php
Comment (johanadivare) [inline src/Rector/MethodCall/ConvertRenderTemplateShortNotationToBundleSyntaxRector.php:25]: Might be nice to add this rule in symfony43.php as its fixing a deprecation there or should it be in symfony50.php as it is removed in there?

File: tests/Rector/MethodCall/ConvertRenderTemplateShortNotationToBundleSyntaxRector/Fixture/test_skip_fixture.php.inc
Comment (johanadivare) [inline tests/Rector/MethodCall/ConvertRenderTemplateShortNotationToBundleSyntaxRector/Fixture/test_skip_fixture.php.inc:16]: With a skip test you can remove the bottom part

**Comment by arnaubuch** [comment]
@johanadivare Applied suggestions, waiting for another review!

File: src/Rector/MethodCall/ConvertRenderTemplateShortNotationToBundleSyntaxRector.php
Comment (samsonasik) [inline src/Rector/MethodCall/ConvertRenderTemplateShortNotationToBundleSyntaxRector.php:82]: First class callable `$this->call(...)` need to be skipped as well.

```suggestion
        if ($node->isFirstClassCallable()) {
               return null;
        }
```

**Comment by samsonasik** [comment]
Please run:

```
vendor/bin/rector && composer fix-cs 
```

to make CI green.

**Review by samsonasik — APPROVED** [review]

File: tests/Rector/MethodCall/ConvertRenderTemplateShortNotationToBundleSyntaxRector/Fixture/test_skip_fixture.php.inc
Comment (JohJohan) [inline tests/Rector/MethodCall/ConvertRenderTemplateShortNotationToBundleSyntaxRector/Fixture/test_skip_fixture.php.inc:7]: Name `skip_already_bundle_name.php.inc` and the controller `SkipAlreadyBundleName`

File: tests/Rector/MethodCall/ConvertRenderTemplateShortNotationToBundleSyntaxRector/Fixture/test_fixture.php.inc
Comment (JohJohan) [inline tests/Rector/MethodCall/ConvertRenderTemplateShortNotationToBundleSyntaxRector/Fixture/test_fixture.php.inc:7]: You could rename the fixture file `shortToBundleName.php.inc` and controller to `ShortToBundleName` to make clear what this fixture is changing, maybe you can find better naming

File: tests/Rector/MethodCall/ConvertRenderTemplateShortNotationToBundleSyntaxRector/Fixture/test_fixture_renderView.php.inc
Comment (JohJohan) [inline tests/Rector/MethodCall/ConvertRenderTemplateShortNotationToBundleSyntaxRector/Fixture/test_fixture_renderView.php.inc:7]: You could rename the fixture file `shortToBundleNameRenderView.php.inc` and controller to `ShortToBundleNameRenderView` to make clear what this fixture is changing, maybe you can find better naming

File: tests/Rector/MethodCall/ConvertRenderTemplateShortNotationToBundleSyntaxRector/Fixture/test_skip_fixture.php.inc
Comment (arnaubuch) [reply tests/Rector/MethodCall/ConvertRenderTemplateShortNotationToBundleSyntaxRector/Fixture/test_skip_fixture.php.inc:7]: Renamed the three tests, hopefully it's clear enough.

**Comment by TomasVotruba** [comment]
Looks good, thank you :+1:

**Pull request merged**
